### PR TITLE
update comparisons in FAQ

### LINKED ISSUE #4123: Notify users of new github web editor
A couple of weeks ago, github added a web based editor. Using it is simple, change the url to github.dev or press `.` in any repo.

While this version of vscode is limited as it has no backend I think it can be useful for quick edits/contributions.

Perhaps let users know they can use it as an alternative if they cannot setup code-server or if they can't deploy it to a cloud provider or aren't able to pay for github codespaces.

I'm not saying it is hard to setup code-server (its very easy actually).

https://docs.github.com/en/codespaces/developing-in-codespaces/web-based-editor

### PR DESCRIPTION
**Pull request by bpmct**
resolves #4123 #4289

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -377,18 +378,31 @@ for extensions.
 
 Theia doesn't allow you to reuse your existing VS Code config.
 
+## What's the difference between code-server and OpenVSCode-Server?
+
+code-server and OpenVSCode-Servr both allow you to access VS Code via a
+browser. The two projects also use their own [forks of VS Code](https://github.com/cdr/vscode) to
+leverage modern VS Code APIs and stay up to date with the upsteam version.
```
Comment (bpmct) [inline docs/FAQ.md:385]: @TeffenEllis - you have much more context here, and it very is possible I am not understanding this correctly. would love your 👀

**Comment by jsjoeio** [comment]
Maybe your editor changed these files? Running `yarn fmt` locally will fix. Happy to help too!

image

File: docs/FAQ.md (hunk 1 above)
Comment (jsjoeio) [reply docs/FAQ.md:385]: This looks good to me (in general!)

File: docs/npm.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
```
Comment (jsjoeio) [inline docs/npm.md:3]: Yeah, I don't think our format script likes these new line breaks 🤣

**Review by jsjoeio — APPROVED** [review]
LGTM! (minus the typos Jonathan pointed out + `yarn fmt` fix)

File: docs/FAQ.md [hunk 3]
```diff
@@ -377,18 +379,31 @@ for extensions.
...
 
 Both code-server and VS Code Codespaces allow you to access VS Code via a
-browser.
-
-VS Code Codespaces, however, is a closed-source, paid service offered by
-Microsoft. While you can self-host environments with VS Code Codespaces, you
-still need an Azure billing account, and you must access VS Code via the
-Codespaces web dashboard instead of connecting directly to it.
+browser. VS Code Codespaces, however, is a closed-source, paid service offered by
+Microsoft. Codespaces are hosted in Microsoft's cloud.
```
Comment (ashumz) [inline docs/FAQ.md:403]:
```suggestion
GitHub. Codespaces can only be purchased through GItHub and are hosted on Microsoft's cloud, Azure.
```

File: docs/FAQ.md [hunk 4]
```diff
@@ -377,18 +379,31 @@ for extensions.
...
 ## What's the difference between code-server and VS Code Codespaces?
 
 Both code-server and VS Code Codespaces allow you to access VS Code via a
-browser.
-
-VS Code Codespaces, however, is a closed-source, paid service offered by
-Microsoft. While you can self-host environments with VS Code Codespaces, you
-still need an Azure billing account, and you must access VS Code via the
-Codespaces web dashboard instead of connecting directly to it.
+browser. VS Code Codespaces, however, is a closed-source, paid service offered by
```
Comment (ashumz) [inline docs/FAQ.md:402]:
```suggestion
browser. GitHub Codespaces, however, is a closed-source, paid service offered by
```

File: docs/FAQ.md [hunk 5]
```diff
@@ -377,18 +379,31 @@ for extensions.
...
+- proxy web ports
+- certificate support
+- plugin API
+- settings sync (coming soon)
+
+For more details, see [this discussion post](https://github.com/cdr/code-server/discussions/4267#discussioncomment-1411583).
+
 ## What's the difference between code-server and VS Code Codespaces?
 
 Both code-server and VS Code Codespaces allow you to access VS Code via a
```
Comment (ashumz) [inline docs/FAQ.md:401]:
```suggestion
Both code-server and GitHub Codespaces allow you to access VS Code via a
```

File: docs/FAQ.md [hunk 6]
```diff
@@ -377,18 +379,31 @@ for extensions.
 
 Theia doesn't allow you to reuse your existing VS Code config.
 
+## What's the difference between code-server and OpenVSCode-Server?
+
+code-server and OpenVSCode-Server both allow you to access VS Code via a
+browser. The two projects also use their own [forks of VS Code](https://github.com/cdr/vscode) to
+leverage modern VS Code APIs and stay up to date with the upsteam version.
+
+However, OpenVSCode-Server is scoped at only making VS Code avalible in the web browser.
```
Comment (ashumz) [inline docs/FAQ.md:388]:
```suggestion
However, OpenVSCode-Server is scoped at only making VS Code available in the web browser.
```
Spelling error

File: docs/FAQ.md [hunk 7]
```diff
@@ -377,18 +379,31 @@ for extensions.
...
+
+- password auth
+- proxy web ports
+- certificate support
+- plugin API
+- settings sync (coming soon)
+
+For more details, see [this discussion post](https://github.com/cdr/code-server/discussions/4267#discussioncomment-1411583).
+
 ## What's the difference between code-server and VS Code Codespaces?
```
Comment (ashumz) [inline docs/FAQ.md:399]:
```suggestion
## What's the difference between code-server and GitHub Codespaces?
```

**Review by TeffenEllis — APPROVED** [review]
This looks like a great addition to the our docs! At the moment I’m out of office for a medical procedure so I’ll defer to @jsjoeio and @code-asher.

**Pull request merged**